tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -200,9 +200,12 @@
         - install-os
         - configure-os
         - run-os
+        - reboot-os
```

Comment:
I guess that reboot is conditional based on something and we wouldn't actually reboot during Adoption, right?

Reply:
it is conditional, but I don't remember what is the condition, so I'm not sure if it would happen during adoption